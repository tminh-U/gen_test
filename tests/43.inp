13
16 20 5 4 14 9 14 12 6 5 14 3 16 
14
6 11
13 13
4 8
8 11
10 10
12 12
10 13
8 13
8 12
11 11
10 12
10 11
5 10
5 12
