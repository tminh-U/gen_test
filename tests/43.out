16
2
16
8
2
2
10
18
9
2
5
4
10
21
